eva-accession-release-automation/gather_clustering_counts/gather_clustering_counts_from_mongo.py

```diff
@@ -0,0 +1,378 @@
+import argparse
+import glob
+import os
+import psycopg2
+from collections import defaultdict
+from datetime import datetime
+
+from ebi_eva_common_pyutils.logger import logging_config
+from ebi_eva_common_pyutils.mongodb import MongoDatabase
+from ebi_eva_common_pyutils.config_utils import get_pg_metadata_uri_for_eva_profile
+from ebi_eva_common_pyutils.pg_utils import execute_query, get_all_results_for_query
+from gather_per_species_clustering_counts import assembly_table_name
+
+
+logger = logging_config.get_logger(__name__)
+logging_config.add_stdout_handler()
+
+
+def gather_count_from_mongo(clustering_dir, mongo_source, private_config_xml_file, release_version):
+    # Assume the directory structure:
+    # clustering_dir --> <scientific_name_taxonomy_id> --> <assembly_accession> --> cluster_<date>.log_dict
+
+    all_log_pattern = os.path.join(clustering_dir, '*', 'GCA_*', 'cluster_*.log')
```

Comment:
Relying on clustering logs to get the date range will not work this time around because there will be some of the clustering done during remapping.

Reply:
[Addressed](https://github.com/EBIvariation/eva-accession/pull/372/commits/159a8552339a1b45db83eada12660c068aeca79b)